src/scss/components/_media-element.scss

```diff
@@ -159,7 +159,7 @@ $mediaElement-spacing           : spacing(x2);
         text-decoration: none;
         top: $mediaElement-spacing;
 
-        @include media('>=narrow') {
+        @include media('<mid') {
```

Comment:
Will this change affect any other components? (Might be used on menu for instance)

Reply:
@ashleynolan - this is a custom media query I made for the last release but asked Insung to change it for me. Sorry for the confusion 👍